app/models/user.rb

```diff
@@ -1,2 +1,8 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
```

Comment:
- [ ] Greate job so far 👏🏻 However you're missing one association between the user model and like models according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). kindly address this ✍🏻 

``has_many :likes, foreign_key: 'author_id'``